### PR TITLE
Updates issue templates

### PR DESCRIPTION
**Pull request by widal001**
## Summary

Updates repository issue templates with the templates found in this [sandbox repository](https://github.com/agilesix/simpler-grants-sandbox/issues) following the discussion on [this issue](https://github.com/HHS/grants-product-and-delivery/issues/282)

Fixes https://github.com/HHS/grants-product-and-delivery/issues/294

### Time to review: __5 mins__

- Updates the following templates:
  - Bug
  - Feature request
  - Internal task
- Adds the following templates:
  - Deliverable
  - Epic
  - ADR
- Adds a default issue type to each template
- Disables the use of blank issues

You can experiment with live versions of these issue templates by clicking "New issue" in [the sandbox repo](https://github.com/agilesix/simpler-grants-sandbox/issues)

Here are screenshots of the various templates:

**Issue picker**
 
Screenshot 2024-10-04 at 10 01 26 AM

**Bug report**

Screenshot 2024-10-04 at 10 02 05 AM

**Feature request**

Screenshot 2024-10-04 at 10 02 34 AM

**Internal - Decision**

Screenshot 2024-10-04 at 10 02 59 AM

**Internal - Deliverable**

Screenshot 2024-10-04 at 10 03 40 AM

**Internal - Epic**

Screenshot 2024-10-04 at 10 04 48 AM

**Internal - Task**

Screenshot 2024-10-04 at 10 06 21 AM

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/1_bug_report.yml [hunk 1]
```diff
@@ -0,0 +1,53 @@
+name: Bug
+description: Report a bug
+title: "[Bug] "
```
Comment (mdragon) [inline .github/ISSUE_TEMPLATE/1_bug_report.yml:3]: I think we agreed to drop these.

File: .github/ISSUE_TEMPLATE/3_internal_decision.yml [hunk 2]
```diff
@@ -0,0 +1,59 @@
+name: Internal - Decision
+description: Describe a decision that needs to be made and recorded
+title: "[ADR] "
```
Comment (mdragon) [inline .github/ISSUE_TEMPLATE/3_internal_decision.yml:3]: There's a label for "docs:adr" that we could apply and not use the [ADR] prefix.

**Review by mdragon — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/1_bug_report.yml (hunk 1 above)
Comment (widal001) [reply .github/ISSUE_TEMPLATE/1_bug_report.yml]: Fixed in [refactor: Drops all title pre-fixes](https://github.com/HHS/simpler-grants-gov/pull/2369/commits/ee23fa1a73749925203c3351a58a0659a98cdc33)

File: .github/ISSUE_TEMPLATE/3_internal_decision.yml (hunk 2 above)
Comment (widal001) [reply .github/ISSUE_TEMPLATE/3_internal_decision.yml]: Good idea! Just added that default label in this commit: [feat: Adds "docs: adr" label](https://github.com/HHS/simpler-grants-gov/pull/2369/commits/a9ee8bc55ceb4f547ead6a23f06327c20941b4fb) after dropping the prefix in this commit: [refactor: Drops all title pre-fixes](https://github.com/HHS/simpler-grants-gov/pull/2369/commits/ee23fa1a73749925203c3351a58a0659a98cdc33)

**Review by mdragon — COMMENTED** [review]
This looks right to me but I'll let @mxk0 give final sign off since he was deeper in this work than I was

**Comment by mxk0** [comment]
Looks great. Thanks @widal001!

**Comment by margaretspring** [comment]
@widal001 Please go ahead. I'm good with @mxk0 's feedback. Please revoke my approval of the ticket as a code review. I don't want to get up that side to the PR review (not sure these non-code issues work well with that review process).

**Review by mdragon — APPROVED** [review]

**Pull request merged**